wkcuber/downsampling.py

```diff
@@ -187,7 +187,17 @@ def downsample(
 
     with get_executor_for_args(args) as executor:
         job_args = []
-        for target_cube_xyz in target_cube_addresses:
+        processed_data_since_last_logging = 0
+        for i, target_cube_xyz in enumerate(target_cube_addresses):
+            use_logging = False
+            wkw_length = source_wkw.header.file_len * source_wkw.header.block_len
+            processed_data_since_last_logging += wkw_length ** 3
+            if (
+                i == 0 or processed_data_since_last_logging > 1024 ** 3
+            ):  # log every gigabyte of processed data
```

Comment:
```suggestion
            ):  # log every gigavoxel of processed data
```

since a voxel can be non-uint8, gigabyte can be imprecise.